pkg/rancher-desktop/config/kubeconfig.js

```diff
@@ -2,11 +2,8 @@
 
 const fs = require('fs');
 const pth = require('path');
-// TODO: Uncomment after PR https://github.com/kubernetes-client/javascript/pull/748 lands
-// const k8s = require('@kubernetes/client-node');
 
-// TODO: Stop using the next module (and delete) after kubernetes-client/javascript/pull/748 lands
-const findHomeDir = require('@/config/findHomeDir');
+const k8s = require('@kubernetes/client-node');
```

Comment:
I wonder why we never translated this to TypeScript. (Please don't do that in this PR, though.)